src/errorHandler/errorHandler.ts

```diff
@@ -1,68 +1,143 @@
-import { GraphQLError } from 'graphql';
-import { ApolloError } from 'apollo-server-errors';
+import {
+  GraphQLError,
+  GraphQLFormattedError,
+  GraphQLErrorOptions,
+} from 'graphql';
+import { unwrapResolverError } from '@apollo/server/errors';
+
+/**
+ * Internally managed error codes.  If a new error is added here,
+ * it should be added to `NO_REPORT_ERRORS` in `sentryPlugin` if
+ * we do not want to report it to sentry.
+ */
+export enum InternalErrorCode {
```

Comment:
nit: can add comments on how we can add new domain error to this library

Reply:
Added this below on CustomGraphQLError